src/server.py

```diff
@@ -148,17 +144,13 @@ def run_results_day_flot_query():
                  'osx10.6',
                  'osx10.7',
                  'osx10.8']
-    db = create_db_connnection()
 
     data_platforms = {}
     for platform in platforms:
-        cursor = db.cursor()
-        cursor.execute("""select DATE(date) as day,sum(result="%s") as failures,count(*) as
-                          totals from testjobs where platform="%s" and date >= "%s" and date <= "%s"
-                          group by day""" % ('testfailed', platform, start_date, end_date))
-
-        query_results = cursor.fetchall()
-
+        query_results = session.query(Testjobs.date.label('day'),
+                                      func.count(Testjobs.result).label("failures"),
```

Comment:
did we change 'testfailed' -> 'failures'?  I don't know where we get the 'testfailed' value for the result field?